src/V3RandomizeMethod.cpp

```diff
@@ -0,0 +1,171 @@
+// -*- mode: C++; c-file-style: "cc-mode" -*-
+//*************************************************************************
+// DESCRIPTION: Verilator: Expression width calculations
+//
+// Code available from: https://verilator.org
+//
+//*************************************************************************
+//
+// Copyright 2003-2020 by Wilson Snyder. This program is free software; you
+// can redistribute it and/or modify it under the terms of either the GNU
+// Lesser General Public License Version 3 or the Perl Artistic License
+// Version 2.0.
+// SPDX-License-Identifier: LGPL-3.0-only OR Artistic-2.0
+
+#include "config_build.h"
+#include "verilatedos.h"
+
+#include "V3RandomizeMethod.h"
+
+//######################################################################
+// Visitor that marks classes needing a randomize() method
+
+class RandomizeMethodMarkVisitor : public AstNVisitor {
+    typedef std::unordered_set<AstClass*> DerivedSet;
+    typedef std::unordered_map<AstClass*, DerivedSet> BaseToDerivedMap;
+
+    BaseToDerivedMap m_baseToDerivedMap;  // Mapping from base classes to classes that extend them
+
```

Comment:
```suggestion

    // METHODS
    VL_DEBUG_FUNC;  // Declare debug()

    // VISITORS
```

and move methods from below up under // METHODS.